solutions/agents/README.md

```diff
@@ -2,11 +2,11 @@
 
 This architecture deploys the following observability agents on a Red Hat OpenShift cluster:
 
-* Log Analysis agent
```

Comment:
Add this back but add (DEPRECATED) after it. Also add it to the button of the list